cfpb/teachers-digital-platform
# urls.py update for release. change tdp to what is documented

Short description explaining the high-level reason for the pull request

## Changes

- Changed urls.py in the following way:
  - 'crt-survey/' changed to 'tool/'
  - 'crt-start/' changed to 'before-you-begin/'

## Testing

- verify the following link works in your local environment:
  - Before cfgov-refresh PR goes through
    - make this change to the  #cfgov-refresh/cfgov/cfgov/urls.py
`flagged_url('TDP_RELEASE',
                r'^practitioner-resources/youth-financial-education/curriculum-review/tool/',  # noqa: E501
                include_if_app_enabled('teachers_digital_platform',
                                       'teachers_digital_platform.tool-urls')),`

`flagged_url('TDP_RELEASE',
            r'^practitioner-resources/youth-financial-education/curriculum-review/before-you-begin/',  # noqa: E501
            include_if_app_enabled('teachers_digital_platform',
                                    'teachers_digital_platform.begin-urls')),`

`flagged_url('TDP_RELEASE',
            r'^practitioner-resources/youth-financial-education/curriculum-review/prototypes/',  # noqa: E501
            include_if_app_enabled('teachers_digital_platform',
                                    'teachers_digital_platform.prototypes-urls')),  # noqa: E501`

  - After cfgov-refresh PR goes through
    - http://localhost:8000/practitioner-resources/youth-financial-education/curriculum-review/before-you-begin/
    - http://localhost:8000/practitioner-resources/youth-financial-education/curriculum-review/tool

- Verify start over with a new review button takes you back to the start page correctly
- Verify begin review takes you to the tool
- Verify print buttons open a new page to the correct location

## Review

- @dcmouyard 

[Preview this PR without the whitespace changes](?w=0)
